### DISCUSSION TITLE
Are there sample CSV files for training data?

### DISCUSSION THREAD

**Discussion by manyam** [h2oai/h2o-llmstudio#102] [Q&A] [answered]
The wording on "Parent Id Column: An optional column specifying the parent id to be used for chained conversations. The value of this column needs to match an additional column with the name id. If provided, the prompt will be concatenated after preceeding parent rows." seems bit ambiguous specially about preceeding parent rows

**Comment by psinger** [comment, marked as answer]
Yes, so actually when you start the GUI, it will generate a sample dataset out of the box, actually it is a parquet file you will find in `data` folder.

You can also check here: https://www.kaggle.com/code/philippsinger/openassistant-conversations-dataset-oasst1?scriptVersionId=127047926

And here: https://github.com/h2oai/h2o-llmstudio/blob/main/app_utils/utils.py#L1888

To see how it is created.
